### PR TITLE
Update TizenRT for SamsungARTIK

### PR DESCRIPTION
**Pull request by sunghan-chang**
1. add new boards, ARTIK055S
2. add new configurations for secure model
3. refactor build/configs stubs for ARTIK
4. add the sss functionality
5. fix some issue on s5j

### CODE REVIEW

**Comment by sunghan-chang** [comment]
Due to getting SamsungARTIK commits, supporting ROMFS is broken. I will update it soon in here.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/artik05x/artik05x_download.sh [hunk 1]
```diff
@@ -78,17 +79,32 @@ download()
         exit 1
     fi
 
+    # check existence of romfs.img if romfs is enabled
+    if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+        pushd ${OS_DIR_PATH} > /dev/null
+        sh ../tools/fs/mkromfsimg.sh
+        if [ ! -f "${FW_DIR_PATH}/romfs.img" ]; then
+            echo "ROMFS image is not present"
+            exit 1
```
Comment (sunghan-chang) [inline build/configs/artik05x/artik05x_download.sh]: Even if CONFIG_FS_ROMFS is enabled, romfs.img can be missed. That is not a problem. User can enable romfs without content. Let's set some romfs variable. And at flash (program) step, we can check that and decide flashing romfs.img.

File: build/configs/artik05x/artik05x_download.sh [hunk 2]
```diff
@@ -78,17 +79,32 @@ download()
...
     # Download all binaries using openocd script
+    OPENOCD_CMD="flash_protect off; \
+                 flash_write bl1 ${FW_DIR_PATH}/bl1.bin; \
+                 flash_protect on; \
+                 flash_write bl2 ${FW_DIR_PATH}/bl2.bin; \
+                 flash_write sssfw ${FW_DIR_PATH}/sssfw.bin; \
+                 flash_write wlanfw ${FW_DIR_PATH}/wlanfw.bin; \
+                 flash_write os ${TIZENRT_BIN}; "
+    if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+    OPENOCD_CMD=${OPENOCD_CMD}"set romfs_partition_enable 1; \
```
Comment (sunghan-chang) [inline build/configs/artik05x/artik05x_download.sh]: in CONFIG_FS_ROMFS != "y" case,
Don't we need to call "set romfs_parition_enable 0"?

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by btheosam** [comment]
Would n't this cause large duplication? would n't artik05x directory change work? or atleast all common artik code inside artik05x directory?

**Comment by sunghan-chang** [comment]
@btheosam Not duplicated. 05x in configs is a common folder for three boards. 053, 053s and 055s have own default configurations sets.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**